src/commands/Tools/Websearch/horoscope.ts

```diff
@@ -0,0 +1,59 @@
+import { SkyraCommand, SkyraCommandOptions } from '@lib/structures/SkyraCommand';
+import { ApplyOptions } from '@skyra/decorators';
+import { fetch, FetchResultTypes, getColor, createPick } from '@utils/util';
+import { MessageEmbed } from 'discord.js';
+import { KlasaMessage } from 'klasa';
+
+@ApplyOptions<SkyraCommandOptions>({
+	cooldown: 10,
+	description: language => language.tget('COMMAND_HOROSCOPE_DESCRIPTION'),
+	extendedHelp: language => language.tget('COMMAND_HOROSCOPE_EXTENDED'),
+	requiredGuildPermissions: ['EMBED_LINKS'],
+	usage: '<sunsign:sunsign> [tomorrow|yesterday|today:default]',
+	usageDelim: ' '
+})
+export default class extends SkyraCommand {
+
+	private readonly kSunSigns = new Set(['capricorn', 'aquarius', 'pisces', 'aries', 'taurus', 'gemini', 'cancer', 'leo', 'virgo', 'libra', 'scorpio', 'sagittarius']);
+	private kRandomSunSign = createPick([...this.kSunSigns]);
+
+	public async init() {
+		this.createCustomResolver('sunsign', (arg, _, message) => {
+			if (this.kSunSigns.has(arg.toLowerCase())) return arg;
```

Comment:
Does the horoscope support signs in case insensitive? Just checking whether or not this is valid

Reply:
oh shit good catch, because it doesn't.